akka-actor/src/main/scala/akka/actor/Extension.scala

```diff
@@ -132,5 +132,5 @@ trait ExtensionIdProvider {
   /**
    * Returns the canonical ExtensionId for this Extension
    */
-  def lookup(): ExtensionId[_ <: Extension]
+  def lookup: ExtensionId[_ <: Extension]
```

Comment:
is this source compatible?

Reply:
Looks like it, with `-Dakka.no.discipline=true` subclasses with parens are still allowed (checked both 2.12 and 2.13)

Reply:
ok, thanks